### PR TITLE
scheduler.dart import cleanup

### PR DESCRIPTION
**Pull request by ricardoamador**
Remove a redundant import statement.

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: app_dart/lib/src/service/scheduler.dart [hunk 1]
```diff
@@ -10,7 +10,6 @@ import 'package:cocoon_service/src/service/build_status_provider.dart';
 import 'package:cocoon_service/src/service/scheduler/policy.dart';
 import 'package:gcloud/db.dart';
 import 'package:github/github.dart' as github;
-import 'package:github/github.dart';
```
[the remark this answers is not in the document]
Comment (ricardoamador) [reply app_dart/lib/src/service/scheduler.dart:13]: Ah let me see. I don't think there should be an issue with that.

[the remark this answers is not in the document]
Comment (ricardoamador) [reply app_dart/lib/src/service/scheduler.dart:13]: Okay sorry, there would be an issue with this. It is defined in at lest two other imports.

[the remark this answers is not in the document]
Comment (keyonghan) [reply app_dart/lib/src/service/scheduler.dart:13]: what's the conflicting class/import?

[the remark this answers is not in the document]
Comment (ricardoamador) [reply app_dart/lib/src/service/scheduler.dart:13]: Hmm, one sec. Hold that thought. I might be wrong here.

[the remark this answers is not in the document]
Comment (ricardoamador) [reply app_dart/lib/src/service/scheduler.dart:13]: Okay I was wrong. Not sure which class I was editing. Fix is here.

**Comment by ricardoamador** [comment]
Closing to open a new pr as there is something wrong with this one. It is not being updated.

**Pull request closed without merging**